__tests__/formats/scssMaps.test.js

```diff
@@ -207,7 +207,7 @@ var dictionary = {
 describe('formats', () => {
   _.each(['scss/map-flat', 'scss/map-deep'], function(key) {
 
-    describe(key, () => {
+    describe(`${key}`, () => {
```

Comment:
?

Reply:
This might be a rule we could disable... There is a lint rule that `describe` blocks must have a string as the title and the linter isn't smart enough to know that those variables are strings...